.github/workflows/ci-pr.yml

```
@@ -5,7 +5,7 @@ on:
     branches: [ main ]
 
 env:
-    NETCORE_VERSION: '5.0.400'
+    NETCORE_VERSION: '6.0.100-preview.7.21379.14'
```

Just missed this one @linkdotnet 
```suggestion
    NETCORE_VERSION: '6.0.100'
```

---

Good catch!